cmd/ethrex_replay/src/cli.rs

```diff
@@ -181,6 +181,20 @@ pub struct EthrexReplayOptions {
         required = false
     )]
     pub verbose: bool,
+    // CAUTION
+    // This flag is used to create a benchmark file that is used by our CI for
+    // updating benchmarks from https://docs.ethrex.xyz/benchmarks/.
+    // Do no remove it under any circumstances, unless you are refactoring how
+    // we do benchmarks in CI.
+    #[cfg(feature = "ci")]
```

Comment:
do we need to have this behind a feature flag? Also, if we need to, maybe I would name it differently. It is used by `ci` but maybe something like `bench` would be more appropriate. But again, no flag seems the best solution IMO

Reply:
Done in [b2a004a](https://github.com/lambdaclass/ethrex/pull/4683/commits/b2a004af1a5a8781a670cbc400267263277ae415) and [fdb0d4e](https://github.com/lambdaclass/ethrex/pull/4683/commits/fdb0d4ed769aeaff172efc93fa7477f748310922)